### PR TITLE
Add a cosmics PCL workflow to `runTheMatrix`

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Title says it all, in preparation to add a FPix LA PCL workflow (to be run in cosmics data-taking)

#### PR validation:

Run successfully ` runTheMatrix.py -l 1005.0 -t 4 -j 8`

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by mmusich** [comment]
assign alca

**Comment by mmusich** [comment]
test parameters:
   * workflow = 1005.0

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by tvami** [comment]
+alca
 * new wf 1005 ran fine

**Comment by sunilUIET** [comment]
+pdmv

**Comment by mmusich** [comment]
sorry @tvami @sunilUIET I realized there was a small mistake, I fixed and force-pushed, now it should be good to go.

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
+alca
 * new wf passed (still)

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -2173,6 +2175,16 @@ def lhegensim2018ml(fragment,howMuch):
...
+steps['TIER0EXPCOSMICSRUN3']=merge([{'--scenario':'cosmics',
+                                     '-s':'RAW2DIGI,L1Reco,RECO,ALCAPRODUCER:@allForExpressCosmics,ENDJOB',
+                                     '-n':1000,
+                                     '--data':'',
+                                     '--process':'RECO',
+                                     '--era':'Run3',
+                                     '--conditions':'auto:run3_data_express',
+                                     '--datatier':'ALCARECO',
+                                     '--eventcontent':'ALCARECO',
+                                     '--customise_commands':'"process.source.eventsToProcess = cms.untracked.VEventRange(\'360336:24:422152-360336:24:422152\')"'},steps['TIER0EXP']])
```
Comment (malbouis) [inline Configuration/PyReleaseValidation/python/relval_steps.py:2187]: @mmusich , sorry if I misunderstood it, but is this line selecting a single event? Is it intended?

Comment (mmusich) [reply Configuration/PyReleaseValidation/python/relval_steps.py:2187]: Yes

**Comment by mmusich** [comment]
@cms-sw/pdmv-l2 @cms-sw/upgrade-l2 can you please review and sign, so that this can be used in https://github.com/cms-sw/cmssw/pull/40664 (after rebasing)?

**Comment by kskovpen** [comment]
+pdmv

**Comment by srimanob** [comment]
+Upgrade

**Comment by perrotta** [comment]
The new workflow 1005 runs. Let me just point out the following warning message that can be found in the log file of step2 for that workflow, as it may be of interest to its potential users:
```
%MSG-e Alignment:  TkAlCaRecoMonitor:ALCARECOTkAlCosmicsInCollisions0TTkAlDQM  01-Feb-2023 18:37:14 CET Run: 360336 Event: 422152
invalid trackcollection encountered!
```

**Comment by perrotta** [comment]
+1

**Comment by mmusich** [comment]
> The new workflow 1005 runs. Let me just point out the following warning message that can be found in the log file of step2 for that workflow, as it may be of interest to its potential users:
>```
>%MSG-e Alignment:  TkAlCaRecoMonitor:ALCARECOTkAlCosmicsInCollisions0TTkAlDQM  01-Feb-2023 18:37:14 CET Run: >360336 Event: 422152
>invalid trackcollection encountered!
>```

thanks, there was a bug indeed, fixed at  https://github.com/cms-sw/cmssw/pull/40686

**Pull request merged**